crates/red_knot_python_semantic/resources/mdtest/import/builtins.md

```diff
@@ -40,7 +40,7 @@ custom_builtin: Custom
 `/typeshed/stdlib/typing_extensions.pyi`:
 
 ```pyi
-def reveal_type(obj, /): ...
+def reveal_type[T](obj: T, /) -> T: ...
```

Comment:
Though updating `reveal_type` to reveal the return type means that every custom signature for it needs to be generic and plumb through the return type correctly. That seems too brittle, so I will probably revert this part.